lib/wda-mac.js

```diff
@@ -392,10 +404,11 @@ class WDAMacServer {
       }
     }
 
-    if (this.isProxyingRemoteWDAMac) {
+    // TODO: do this.isProxyReady everytime
```

Comment:
already done?

Reply:
I haven't tested well and finished implementation, so just here keeps todo as my next time :)
(And will tweak other lines as well)